### ISSUE TITLE
Add drop down list to the navigation menu for each course group

### ISSUE THREAD

**Issue by noabarlia** [ScDor/Closure#63]
Add drop down list to the navigation menu for each course group, according to the current track.

**Comment by ScDor** [comment]
Cool idea, thanks

future devs, please note adding `supplementary` might be too much (as there
are ~8500 courses, most of which are supplementary for most tracks), and I
believe does not provide too much added value to the users (who chan just
search for the courses manually)

**Comment by noabarlia** [comment]
Yes, thanks for the heads up. It is especially useful for 'must', 'choose from list' and 'choose'.
As user I sometimes prefer to see the option in front of me rather then searching manually.
